modules/015-admission-policy-engine/crds/doc-ru-security-policy.yaml

```
@@ -66,116 +66,117 @@ spec:
+                          description: Правило определения значений `supplementalGroups`.
                         ranges:
-                          description: "Список диапазонов ID для `supplementalGroups`, доступных к использованию в режиме `MustRunAs`."
+                          description: Список диапазонов идентификаторов для `supplementalGroups`, доступных режиме `MustRunAs`.
                           items:
                             properties:
                               min:
-                                description: "Минимальное значение диапазона."
+                                description: Минимальное значение диапазона.
                               max:
-                                description: "Максимальное значение диапазона."
+                                description: Максимальное значение диапазона.
                     readOnlyRootFilesystem:
-                      description: "Определяет необходимость запуска контейнеров с корневой файловой системой, доступной только для чтения"
+                      description: |
+                      Если установлен в `true`, разрешает запуск только тех подов, у всех контейнеров которых корневая файловая система доступна только для чтения (в поле `securityContext` установлено `readOnlyRootFilesystem: true`). Подробнее в [документации Kubernetes](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.29/#securitycontext-v1-core).
                     automountServiceAccountToken:
-                      description: "Определяет возможность для подов включать автоматическое монтирование токена сервисного аккаунта"
+                      description: Разрешает запускать поды с автоматическим монтированием токена сервисного аккаунта.
                     seccompProfiles:
-                      description: "Определяет список доступных к использованию seccomp-профилей."
+                      description: Задает список доступных seccomp-профилей.
                       properties:
                         allowedProfiles:
-                          description: "Список seccomp-профилей, доступных к использованию."
+                          description: Список seccomp-профилей.
                         allowedLocalhostFiles:
                           description: |
-                            Определяет локальные профили seccomp (в формате JSON), доступные к использованию, в случае указания `Localhost` в поле `allowedProfiles`.
+                            Определяет локальные профили seccomp (в формате JSON), доступные после указания `Localhost` в поле `allowedProfiles`.
 
-                            Пустой список запрещает использование любых локальных профилей.
+                            Пустой список запрещает использование локальных профилей.
                     seLinux:
-                      description: "Определяет допустимые к использованию значения SElinux-меток."
+                      description: Задает доступные значения SElinux-меток.
```

```suggestion
                      description: Задает доступные значения SElinux-меток.
```

Мне кажется тут стало менее понятно. В этом и подобных параметрах, где убрали "доступных к.".  Я бы это согласовал с Юрой например. 
@MariaMakeeva

---

уже несколько раз обсудили в Slack. закрываю